ui/partials/main/jobs.html

```diff
@@ -16,8 +16,10 @@
     <span class="result-set-left">
       <span class="result-set-title-left">
         <span>
-          <a href="{{::revisionResultsetFilterUrl}}"
+          <a ng-href="{{::revisionResultsetFilterUrl}}"
+             ng-click="changeResultSet(repoName, resultset.revision)"
              title="View only this resultset"
+             prevent-default-on-left-click
```

Comment:
Unfortunately, this prevents the user ``cmd-click``ing the link to open it in a new tab.  That was the purpose behind ``revisionResultsetFilterUrl``

This makes it a bit trickier.  you'd need to either update that scope variable, or find a way to hijack it in the ``ng-click`` to add the filter params.  Tbh, I'm not quite sure how one would do the latter.  

I wonder if there's a way in the ``ng-href`` to call a function to get the href?